icinga2/postgresql-server.sls

```diff
@@ -1,4 +1,6 @@
-{% if salt['pillar.get']('icinga2:postgres:use_formula', False) %}
+{% from "icinga2/map.jinja" import icinga2 with context %}
```

Comment:
This line was absolutely needed.  I also noticed this in the `semantic-release` PR: dece815.